src/lobby/controller.js

```diff
@@ -210,6 +203,10 @@ function LobbyController(CardshifterServerAPI, $scope, $timeout, $rootScope, $lo
     function formatTimeNumber(time) {
         return time < 10 ? "0" + time : time;
```

Comment:
Well, momentjs also includes localisation - unless there is no interest in localising Cardshifter?